tests/test_bookmark.py

```diff
@@ -1,39 +1,60 @@
-import unittest
 from base import CloseioBase
 from tap_tester.base_suite_tests.bookmark_test import BookmarkTest
 
+
 class CloseioBookmarkTest(BookmarkTest, CloseioBase):
     """Closeio bookmark test implementation"""
 
+    bookmark_format = "%Y-%m-%dT%H:%M:%S.%f+00:00"
+    initial_bookmarks = {
+            'bookmarks': {
+                'custom_fields': {'date_updated': '2016-07-07T14:22:04.624000Z'},
+                'leads': {'date_updated': '2020-05-06T18:54:08.250000Z'},
+                'activities': {'date_created': '2020-11-06T20:17:17.454000Z'},
```

Comment:
Line 59 mentions the `activities` stream having a different bookmark format that is missing the fractional seconds. Should we change it to match that format here?